subsys/net/ip/tcp.c

```diff
@@ -369,15 +369,6 @@ static inline u32_t get_recv_wnd(struct net_tcp *tcp)
 	return min(NET_TCP_MAX_WIN, NET_TCP_BUF_MAX_LEN);
 }
 
-/* True if the (signed!) difference "seq1 - seq2" is positive and less
- * than 2^29.  That is, seq1 is "after" seq2.
- */
-static inline bool seq_greater(u32_t seq1, u32_t seq2)
```

Comment:
I didn't think that it would be easier for C programmers... For Java, maybe but for C? How do they work with qsort() then? But of course, we can have helpers. How many of them - all 6 (eq, neq, lesser, greater, lessereq, greatereq), or just subset?